stable/coscale/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: coscale
+version: 3.9.1
```

Comment:
The version of CoScale should be in the `appVersion` field, `version` is the chart version and should be `0.1.0`.